modules/clightning/module.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
+#include <span>
+
+#include "module.h"
+#include "clightning_lib.h"
+
+namespace bitcoinfuzz
+{
+    namespace module
+    {
+        CLightning::CLightning(void) : BaseModule("CLightning") {}
+
+        std::optional<std::string> CLightning::deserialize_invoice(std::string str) const
+        {
+            return clightning_des_invoice(str.c_str());
```

Comment:
It looks like there's no need to have a separate `clightning_lib.cpp` file anymore.  We can put the definition of `clightning_des_invoice` in this file instead.